_src/pattern-library/sass/utilities/_helpers.scss

```diff
@@ -122,7 +122,7 @@
     );
 
     border-bottom: 1px solid $transparent;
-    color: palette(primary, base);
+    color: palette(primary, dark);
```

Comment:
The base color is fine on white, but it barely fails on gray. I darkened this a bit, which is suitable on both white or gray.